wallets/provider-tron-link/src/index.ts

```diff
@@ -59,14 +102,14 @@ export const subscribe: Subscribe = ({ updateAccounts, disconnect }) => {
     }
   });
 };
-
-export const canSwitchNetworkTo: CanSwitchNetwork = () => false;
+export const canSwitchNetworkTo: CanSwitchNetwork = canSwitchNetworkToEvm;
 
 export const getSigners: (provider: any) => SignerFactory = signer;
 
 export const getWalletInfo: (allBlockChains: BlockchainMeta[]) => WalletInfo = (
   allBlockChains
 ) => {
+  const evms = evmBlockchains(allBlockChains);
```

Comment:
It returns the whole list evm compatible networks, but we only need BSC and ETH here as the wallet only has support for this.